tests/sentry/api/endpoints/test_project_platforms.py

```diff
@@ -12,4 +12,4 @@ def test_simple(self):
         url = '/api/0/projects/{}/{}/platforms/'.format(project.organization.slug, project.slug)
         response = self.client.get(url, format='json')
         assert response.status_code == 200, response.content
-        assert response.data[0]['platform'] == pp1.platform
```

Comment:
@denamwangi 
Are you exepecting more than one project_platform typically? I had thought it was just 1-to-1.